pages/_app.tsx

```diff
@@ -89,6 +89,7 @@ function MyApp({ Component, pageProps }: AppProps) {
         redirectUri={process.env.APP_URL}
         cacheLocation={"localstorage"}
         audience={"urn:plant-for-the-planet"}
+        useRefreshTokens={true}
```

Comment:
Is this something we should also do at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/561a45aacae1529254f34b46ad75ff5d0040dcfd/pages/_app.tsx#L172?